shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,12 +36,18 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The view displaying Flutter content. May return |nil|, for instance in a headless environment.
+ * The default view displaying Flutter content.
  *
- * WARNING: If/when multiple Flutter views within the same application are supported (#30701), this
- * API will change.
+ * This method may return |nil|, for instance in a headless environment.
+ *
+ * The default view is a special view operated by single-view APIs.
+ */
+- (nullable NSView*)view;
```

Comment:
I don't know if it's the same as the internal breakage, but this broke `file_selector_macos`, and would potentially break any other Swift plugin as well. In Obj-C the same syntax can be used to call methods and access properties, but that's not the case in Swift, so changing a property to a method is a breaking change, which would make supporting both `master` and `stable` with a single plugin impossible. We should not change this without a *very* compelling reason.

Reply:
Yes, this is exactly why this PR was reverted. I did this only because I thought it was equivalent (for objC). I'll change it back to proprty syntax.